setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = apig-wsgi
-version = 2.4.0
+version = 2.4.1
```

Comment:
Bumping version number is only done at release - in general I don't think it's helpful to include it in PR's to most projects

Reply:
Yep my bad.